.circleci/config.yml

```diff
@@ -128,9 +132,6 @@ commands:
             source $BASH_ENV
             source $HOME/venv/bin/activate
 
-            # convert boolean parameter to an env var storing 0 or 1
-            USE_DOUBLE=$(( 0 <<# parameters.usedouble >> + 1 <</ parameters.usedouble >> ))
```

Comment:
I don't totally follow why this we no longer need to set USE_DOUBLE... (being lazy -- I haven't looked carefully but this does seem to be used in the answer_tests).

Reply:
It turns out we weren't actually using this variable for anything. We are directly using `parameters.usedouble` in the call to `cmake` (see line 141 of this file)

Reply:
But doesn't this set an environment variable which is (potentially) used on line 14 of test/answer_tests/test_grackle.py?

Reply:
Right, that's true! But there's a subtlety here. 

Each job in our CI is currently designed so that each job is composed of a sequence of reusable commands:
- Environment variables are not carried over from one command to the next. 
- Tangentially, this is why lots of commands begin with `source $BASH_ENV` - it restores a restore a set of environment variables from a file that we wrote to disk in the "set-env" command
-  The highlighted snippet was part of a command that builds enzo-e
- the `pytest` framework is invoked in a separate command. In that command ([it starts here](https://github.com/enzo-project/enzo-e/blob/642d7d7e58ba5f9dd78c107ff2d6f861621aa8ce/.circleci/config.yml#L212)), we set up the necessary environment variables before invoking `pytest`

Reply:
Ah, OK! Thanks, that explain a number of things.